### DISCUSSION TITLE
Copilot Malfunctioning

### DISCUSSION THREAD

**Discussion by TheFarawayDev** [community/community#122565] [Copilot Conversations]
### Topic Category: Bug

When operating the inline version it fails to operate and outputs "Invalid GAP" which leads me to believe that a new update or change in things has caused a failure in the system even though I barely use it, it is important if I can't fix something quickly. As well I have found previous articles on how to fix it [here](https://github.com/microsoft/vscode-copilot-release/issues/1034) . But mostly it is extra work that isn't needed or it doesn't work so for now I'm using the chat system until fixed.

**Comment by CberYellowstone** [comment]
same on me, `unhandled status from server: 400 bad request: invalid token: missing value` , both VSC and CLion

**Comment by spironoo** [comment]
As of just over an hour or so ago I seem to be having the same problem. IntelliJ and VSCode both.

**Comment by rossoj85** [comment]
Same here.

**Comment by spironoo** [comment]
did this resolve for anyone?

**Reply by CberYellowstone** [reply]
I found my temporary solution: use mobile data (so may this be related to the network?)

**Reply by spironoo** [reply]
that's very abnormal. i'm located in north england, if you're somewhere similar it may be a routing issue?

**Reply by CberYellowstone** [reply]
Yes, it's strange. By the way, is your error message the same as mine?

**Reply by spironoo** [reply]
it is, yes. are you also located in the same area?

**Reply by CberYellowstone** [reply]
Emmm. I'm in China, and for reasons you know, my actual network exit point has tried San Francisco DigitalOcean, Singapore DigitalOcean, Japan AWS, but none of these worked. But when I switch to a cellular network, even at the same network exit, the fault no longer exists, which is very strange.

**Reply by spironoo** [reply]
yeah this makes no sense in that case. i'm all out of ideas

**Reply by CberYellowstone** [reply]
Perhaps you could also try switching to a different network, haha
